simulator-core/src/test/java/com/evbox/everon/ocpp/functional/authorization/EvDriverAuthorizationUsingRfidTest.java

```diff
@@ -0,0 +1,63 @@
+package com.evbox.everon.ocpp.functional.authorization;
+
+import com.evbox.everon.ocpp.simulator.station.actions.Plug;
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.Authorize;
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.StatusNotification;
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.TransactionEvent;
+import com.evbox.everon.ocpp.testutils.station.StationSimulatorSetUp;
+import org.junit.jupiter.api.Test;
+
+import static com.evbox.everon.ocpp.testutils.constants.StationConstants.*;
+import static com.evbox.everon.ocpp.v20.message.common.IdToken.Type.ISO_14443;
+import static com.evbox.everon.ocpp.v20.message.station.StatusNotificationRequest.ConnectorStatus.OCCUPIED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionData.ChargingState.CHARGING;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionData.ChargingState.EV_DETECTED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.EventType.STARTED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.EventType.UPDATED;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.CABLE_PLUGGED_IN;
+import static com.evbox.everon.ocpp.v20.message.station.TransactionEventRequest.TriggerReason.CHARGING_STATE_CHANGED;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class EvDriverAuthorizationUsingRfidTest extends StationSimulatorSetUp {
```

Comment:
Would it make sense to use DisplayName everywhere with specific title of the use-case from OCPP spec?

Reply:
Well, I prefer to have the class names be the chapters, because then our packages/classes match 1:1 with OCPP specs, and you can easily tell what is missing

<img width="359" alt="Screenshot 2019-04-02 at 10 59 28" src="https://user-images.githubusercontent.com/47893389/55389890-686bc900-5536-11e9-98fd-9769df833bf7.png">